src/schema/v2/notification_preferences.ts

```diff
@@ -29,15 +30,23 @@ const subGroupFields = {
     type: new GraphQLNonNull(GraphQLString),
   },
   channel: {
-    type: new GraphQLNonNull(GraphQLString),
+    type: new GraphQLNonNull(
+      new GraphQLEnumType({
+        name: "NotificationChannel",
+        values: {
+          EMAIL: { value: "EMAIL" },
+          PUSH: { value: "PUSH" },
+        },
+      })
+    ),
   },
   status: {
     type: new GraphQLNonNull(
       new GraphQLEnumType({
         name: "SubGroupStatus",
         values: {
-          SUBSCRIBED: { value: "Subscribed" },
-          UNSUBSCRIBED: { value: "Unsubscribed" },
+          SUBSCRIBED: { value: "SUBSCRIBED" },
+          UNSUBSCRIBED: { value: "UNSUBSCRIBED" },
```

Comment:
Similarly what effect does changing these values have?

Reply:
Values can be anything. If these were used by something else then this would likely constitute a breaking change.

Reply:
there isn't anywhere else in the MP schema that these values are being used outside of notification preferences, so I'm not following?

Reply:
There's a mutation that uses this already in the `PreferencesApp` — changing the value here is going to change what Gravity receives for those.

Reply:
That might be fine — it also might not be fine. I do not know, you would have to trace this. But also why bother changing it? GraphQL isn't yelling about values.

Reply:
it was definitely yelling for me, but worth taking another look on the case sensitivity issues maybe not really being an issue before this gets merged

Reply:
graphql would say things like, got "Subscribed" but expected "SUBSCRIBED"

Reply:
That's more likely your spec.

Reply:
my most recent commits actually solves the issues with case transforming the enum types in the specs and mutation/resolver code. It came up as an error in test queries to MP locally so definitely made sense to take another look and fix. I believe this can be resolved now